### PR TITLE
docs: add document section for firestore.settings.cacheSizeBytes

### PR DESCRIPTION
**Pull request by mgenov**
The cacheSizeBytes property documentation is now added in the Settings section of the firestore reference.

### CODE REVIEW

**Comment by mikehardy** [comment]
This looks perfect to start - but we should wait for resolution on how to type the disable-gc constant on the linked code PR so we can document it here

**Comment by mikehardy** [comment]
pasting this here for reference text content once "unlimited" item is handled in code PR: https://firebase.google.com/docs/reference/android/com/google/firebase/firestore/FirebaseFirestoreSettings.html#CACHE_SIZE_UNLIMITED

**Comment by mgenov** [comment]
Added a note for the disabling of GC.

File: docs/firestore/reference/Settings.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ The hostname to connect to.
 
 Whether to use persistence.
 
+### cacheSizeBytes
+[method]cacheSizeBytes returns nullable number;[/method]
+
+Sets an approximate cache size threshold for the on-disk data. If the cache grows beyond this size, Firestore will start removing data that hasn't been recently used. The size is not a guarantee that the cache will stay below that size, only that if the cache exceeds the given size, cleanup will be attempted.
+
```
Comment (Salakar) [inline docs/firestore/reference/Settings.md]: Should we mention the default size of 100mb and any min / max sizes to match the validation?

Comment (mikehardy) [reply docs/firestore/reference/Settings.md]: A good idea, I have some other doc updates to do before release, I'll handle it from here @mgenov has done excellent work by all accounts and I'll be in the docs anyway

Comment (mgenov) [reply docs/firestore/reference/Settings.md]: Added parapgraph for the default size, min and unlimited.

**Pull request merged**